src/main/resources/application.yml

```diff
@@ -10,7 +10,7 @@ otel:
     exporter: otlp
   exporter:
     otlp:
-      endpoint: https://otlp.nr-data.net:4317
+      endpoint: https://otlp.eu01.nr-data.net:4317
```

Comment:
NOTE. Need this change of URL if your New Relic account is setup for EU instead of US region